sdk/core/AzureCore/Source/ClientLoggerProtocol.swift

```diff
@@ -60,42 +65,67 @@ extension ClientLogger {
     }
 }
 
+extension ClientLoggerProtocol {
+    static func defaultTag() -> String {
```

Comment:
I agree, but the most common place this would actually surface would be in tests, which won't matter. The AppConfig and Storage client libraries actually specify a tag, which means this code will not be triggered.